lib/src/test/modular_test_interface.dart

```diff
@@ -0,0 +1,82 @@
+// import '../../flutter_modular.dart';
+
+import 'package:flutter/material.dart';
+
+import '../../flutter_modular.dart';
+import 'utils_test.dart';
+
+enum ModularTestType { resetModule, keepModulesOnMemory }
+
+abstract class IModularTest {
+  final ModularTestType modularTestType;
+  IModularTest({this.modularTestType: ModularTestType.resetModule});
+
+  ChildModule module();
+  List<Bind> binds();
```

Comment:
Just so it doesn't look different from what is done in
MainModule and ChildModule, could you put binds to be a getter instead of a method?

Reply:
https://github.com/Flutterando/modular/blob/df542bce168964d0d34ba3bf73de6f83676700b7/test/app/app_module_test_modular.dart#L17-L28

It makes sense for me. I have just changed the three basic methods and the code stayed cleaner.

Reply:
Perfect!
LGTM!!